test/fakes/article_service.rb

```diff
@@ -24,10 +24,6 @@ def initialize(*)
       @called_methods = { scopes: {}, filters: {}, batch_actions: {}, ordering: {} }
     end
 
-    def resource_class
```

Comment:
This one kinda made the current test a bit obsolete.. We should fix that even if we don't merge this PR